pset-iterators.js

```diff
@@ -65,7 +69,12 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 2:')
 
 // Add your code below this line
+const presidentNamedJames = presidents.filter((nameJames) => {
```

Comment:
@cbrooks2019  nice approach here,  everything works as expected, however you didn't need the temporary variable `result`

instead you could have just written the following in your callback and it would have accomplished the same thing:

```
return nameJames.president.startsWith("James");
```